db/pager_unix.go

```diff
@@ -106,7 +108,7 @@ func (f *filePager) CheckReservedLock() (bool, error) {
 	lock := &unix.Flock_t{
 		Type:   unix.F_WRLCK,
 		Whence: seek_set,
-		Start:  sqlite_shared_first,
+		Start:  sqlite_reserved_byte,
```

Comment:
This looked like an unintentional typo from my reading of sqlite3.c, I can remove the change if I misunderstood.

Reply:
Agree, must have been a copy from another Flock. Nice find!